ocs_ci/helpers/helpers.py

```
@@ -7669,3 +7677,128 @@ def delete_cephfs_ec_pool(pool_name):
+
+    Args:
+        pod_obj (Pod): Pod to exec into
+        file_path (str): Absolute path to the file
+        expected_uid (int): Expected numeric UID
+        expected_gid (int): Expected numeric GID
+
+    Returns:
+        tuple: ``(uid, gid)`` parsed from ``ls -ln`` output
+    """
+    out = pod_obj.exec_sh_cmd_on_pod(f"ls -ln {file_path}")
+    logger.info(f"ls -ln {file_path}: {out}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)